Cargo.toml

```diff
@@ -23,6 +23,7 @@ tokio-native-tls = ["tokio-runtime", "real-tokio-native-tls", "real-native-tls",
 tokio-rustls-webpki-roots = ["tokio-runtime", "real-tokio-rustls", "webpki-roots", "tungstenite/rustls-tls"]
 tokio-rustls-native-certs = ["tokio-runtime", "real-tokio-rustls", "rustls-native-certs", "tungstenite/rustls-tls"]
 tokio-openssl = ["tokio-runtime", "real-tokio-openssl", "openssl"]
+no-verbose-logging = []
```

Comment:
It probably makes sense to reverse this: add a `verbose-logging` feature and by default don't have those verbose logs.

Reply:
I picked `no-verbose-logging` to keep the original behavior. But nobody heavily rely on these verbose log, right?

Reply:
Well, I am sometimes but I can also enable it then :)